djangocms_url_manager/compat.py

```diff
@@ -1,11 +1,9 @@
-from distutils.version import LooseVersion
+import django
 
-import cms
+from packaging.version import Version
 
 
-CMS_VERSION = cms.__version__
-
-CMS_36 = LooseVersion(CMS_VERSION) < LooseVersion("3.7")
+DJANGO_4_2 = Version(django.get_version()) >= Version('4.2') and Version(django.get_version()) < Version('4.3')
```

Comment:
```suggestion
DJANGO_4_2 = Version(django.get_version()) < Version('4.3')
```
We have the convention that `DJANGO_4_2` implies: "Django versions up to 4.2". This seems to be the way you are using it, too.